pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
+
+`sudo modprobe zram num_devices={{2}}`
+
+- Initialise the next free zram device to a 2GB virtual drive:
+
+`sudo zramctl --find --streams {{compression_streams_count}} --size {{2GB}} --algorithm lz4`
```

Comment:
lz4 to be in tokens. And also, since you have already given a concrete no. for size, maybe you should also give a concrete no. for "compression_streams_count".